webapp/src/js/services/db-sync.js

```diff
@@ -129,6 +141,11 @@ angular
 
             syncIsRecent = true;
             sendUpdate({ aggregateReplicationStatus: 'not_required' });
+
+            return DB().info()
+            .then(dbInfo => {
```

Comment:
This is a minor thing but it keeps happening... the rest of the codebase is formatted so that this line is indented two spaces more. eg:

```js
return DB().info()
  .then(dbInfo => {
```

If this style is something you feel strongly about then we should discuss it and update the style guide and start switching the code over to this style.

Personally I find your way hard to read because it looks like a separate statement, rather than something that's working on the return of the first line.

Reply:
Changed